test/fixtures/dbt_project/dbt_project.yml

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
What do you think about paring this down to the minimum required dbt_profile.yml to test the project?

Reply:
Totally! I'll try to make it as slim as possible